specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/Diagnostics.json

```diff
@@ -1681,9 +1681,13 @@
               "description": "metadata for the detector"
             },
             "dataset": {
-              "$ref": "#/definitions/DiagnosticData",
-              "description": "Data Set"
+              "type": "array",
+              "description": "Data Set",
+              "items": {
+                "$ref": "#/definitions/Solution"
```

Comment:
Why did you change the item type from `DiagnosticData` to `Solution`?

Reply:
Copy paste error, corrected.